pycbc/coordinates/base.py

```diff
@@ -14,7 +14,7 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-""" Coordinate transformations.
+""" Base coordinate transformations.
```

Comment:
I'm not sure this really clarifies anything? What is meant here? What kind of coordinate transformations are in this module?